doc/fields.md

```diff
@@ -0,0 +1,90 @@
+Email Alert API Field Reference
+===============================
+
+## Subscriber Lists
+
+Subscriber Lists in Email Alert API can have tags or links. When a document is
+is changed, the API does a search for the document's attributes in the API to
+find a matching subscriber list.
+
+### Tags
+
+The API has a whitelist of available tags that a user can subscribe to.
+
+The following tags are permitted:
```

Comment:
I wonder if it's better to link to the code here because I suspect it'll be easy for this list to go out of the date with the real list.

Reply:
Added that, thanks. I like the approach Search API has (generating https://docs.publishing.service.gov.uk/apis/search/fields.html from the current list), so might be nice to do the same.